src/BuildScriptGenerator/BuildScriptGenerator.csproj

```diff
@@ -26,7 +26,7 @@
     <PackageReference Include="Microsoft.Extensions.Options" Version="2.2.0" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
     <PackageReference Include="Scriban.Signed" Version="1.2.9" />
-    <PackageReference Include="SemanticVersioning" Version="1.2.0" />
+    <PackageReference Include="SemanticVersioning" Version="1.2.2" />
```

Comment:
Upgrading here because the latest version has some bug fixes related to handling semantic versioning ranges